notebooks/official/automl/sdk_automl_tabular_forecasting_batch.ipynb

```diff
@@ -0,0 +1,1056 @@
+{
```

Comment:
<p>Should we mention prediction service here given we don't use it in the notebook and Forecasting doesn't support online prediction?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/84/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Very good point! Will remove.</p>